src/Redux/Categories/categories-redux.js

```diff
@@ -0,0 +1,17 @@
+const CHECK_STATUS = 'bookstore/src/redux/categories/checkStatus';
```

Comment:
- **[OPTIONAL]** I think this file name should be just `categories.js` since its parent folder `src/redux` gives it enough context already.